lib/Platform/Intl/PlatformIntlICU.cpp

```diff
@@ -591,35 +362,31 @@ vm::ExecutionStatus DateTimeFormatICU::initialize(
   // 17. Let r be ResolveLocale(%DateTimeFormat%.[[AvailableLocales]],
   // requestedLocales, opt, %DateTimeFormat%.[[RelevantExtensionKeys]],
   // localeData).
-  static constexpr std::u16string_view relevantExtensionKeys[] = {
+  static std::unordered_set<std::u16string> relevantExtensionKeys{
```

Comment:
Could we use a constexpr array here and below, similar to the apple implementation?

Reply:
Done in 3rd commit.